eng/illink.targets

```diff
@@ -31,6 +38,8 @@
     <ILLinkTrimXml Condition="'$(ILLinkTrimXml)' == '' and Exists('$(MSBuildProjectDirectory)/ILLinkTrim.xml')">$(MSBuildProjectDirectory)/ILLinkTrim.xml</ILLinkTrimXml>
     <!-- ILLinkTrim_LibraryBuild.xml files are only used during building the library, not an app. They shouldn't be embedded into the assembly. -->
     <ILLinkTrimXmlLibraryBuild Condition="'$(ILLinkTrimXmlLibraryBuild)' == '' and Exists('$(MSBuildProjectDirectory)/ILLinkTrim_LibraryBuild.xml')">$(MSBuildProjectDirectory)/ILLinkTrim_LibraryBuild.xml</ILLinkTrimXmlLibraryBuild>
+    <ILLinkSubstitutionsXml Condition="'$(ILLinkSubstitutionsXml)' == '' and Exists('$(MSBuildProjectDirectory)/ILLink.Substitutions.xml')">$(MSBuildProjectDirectory)/ILLink.Substitutions.xml</ILLinkSubstitutionsXml>
```

Comment:
NIT: Feels a bit weird to have a property and item with the same name for the same purpose, do you actually need the property or can you just use the item?

Reply:
We need the property, because there can be only 1 - the linker respects a single file named `ILLink.Substitutions.xml` in the assembly.

I can rename one of them, if you have good suggestions for names.

Reply:
I guess it's fine, perhaps the only very small change that I would make (you don't have to if you don't like it) is to make the item one plural to kind of indicate when to use which: `ILLinkSubstitutionsXmls` item when there are more than one and `ILLinkSubstitutionsXml` property when there is only one.

Reply:
I can do that. Thanks